packages/trader/src/App/Components/Elements/market-is-closed-overlay.tsx

```diff
@@ -42,12 +45,14 @@ const MarketIsClosedOverlay = ({
             <Text as='p' className='market-is-closed-overlay__main-heading' color='prominent' weight='bold'>
                 <Localize i18n_default_text='This market is closed' />
             </Text>
-            <MarketCountdownTimer
-                is_main_page
-                setIsTimerLoading={setIsTimerLoading}
-                onMarketOpen={onMarketOpen}
-                symbol={symbol}
-            />
+            <React.Suspense fallback={<div />}>
```

Comment:
I think we can use <UILoader/> instead of div for fallback

Reply:
yes, we should have UILoader instead of div as fallback